.appveyor.yml

```diff
@@ -21,46 +23,30 @@ pull_requests:
 
 os: Visual Studio 2019
 
-init:
-  - git config --global core.autocrlf input
```

Comment:
Why was this removed?

Reply:
It doesn't do anything. It's a leftover from the appveyor yml reference: https://www.appveyor.com/docs/appveyor-yml/